addons/hr/models/hr_employee_base.py

```diff
@@ -305,12 +305,13 @@ def _get_employee_working_now(self):
                     working_now += res_employee_ids.ids
         return working_now
 
-    def _get_calendar_periods(self, start, stop):
```

Comment:
I would rather you don't change signature for the method :innocent: 

Instead you can indicate it is docstring that we expect datetime for these arguments